glyphs-reader/src/font.rs

```diff
@@ -1630,23 +1620,26 @@ impl TryFrom<RawLayer> for Layer {
     }
 }
 
-impl TryFrom<RawGlyph> for Glyph {
-    type Error = Error;
-
-    fn try_from(from: RawGlyph) -> Result<Self, Self::Error> {
+impl RawGlyph {
+    // we pass in the radix because it depends on the version, stored in the font struct
+    fn cook(self, codepoint_radix: u32) -> Result<Glyph, Error> {
```

Comment:
I don't find "cook" overly intuitive, `to_glyph` perhaps ?

Reply:
for non-copy types, clippy expects `to_` methods to take self by reference. Can go with 'build', which is maybe clearer?

fwiw the idea of a 'cook' or 'bake' method to convert a 'raw' type into its non-raw equivalent is something I've seen elsewhere. Given that this is an internal method that's only called in one place I don't think we need to stress too much..